quinn/src/connection.rs

```diff
@@ -566,6 +617,10 @@ impl SendStream {
     pub fn poll_write(&mut self, buf: &[u8]) -> Poll<usize, WriteError> {
         use crate::quinn::WriteError::*;
         let mut conn = self.conn.lock().unwrap();
+        if self.is_0rtt {
+            conn.check_0rtt()
+                .map_err(|()| WriteError::ZeroRttRejected)?;
+        }
```

Comment:
I think we need this on `poll_finish` too.

Reply:
Yup, fixed it up.